Assets/LeapMotion/Modules/InteractionEngine/Scripts/InteractionController.cs

```diff
@@ -196,6 +196,21 @@ public abstract class InteractionController : MonoBehaviour,
     /// </summary>
     public Action<InteractionBehaviour> OnStayPrimaryHoveringObject = (intObj) => { };
 
+    /// <summary>
+    /// Called when the InteractionController begins grasping an object.
+    /// </summary>
+    public Action OnGraspBegin = () => { };
```

Comment:
It doesn't look like these are called properly during a SwapGrasp

Reply:
will fix